images/router/haproxy/conf/haproxy-config.template

```diff
@@ -39,17 +39,34 @@ defaults
   option httplog
   log global
 {{ end }}
+
+{{ if (matchPattern "[1-9][0-9]*(us|ms|s|m|h|d)?" (env "ROUTER_DEFAULT_CONNECT_TIMEOUT" "")) }}
```

Comment:
I agree, but the original PR that added the validation only had: 

`{{` if (matchPattern "[1-9][0-9]*(us|ms|s|m|h|d)?" (env "ROUTER_SLOWLORIS_TIMEOUT" "")) }}

So it too would have that problem. However with that said, the `else` clauses just under this set default values when failure occur.